packages/@stylexjs/postcss-plugin/src/bundler.js

```diff
@@ -25,6 +32,7 @@ module.exports = function createBundler() {
         filename: id,
         caller: {
           name: '@stylexjs/postcss-plugin',
+          platform: 'web',
```

Comment:
https://facebook.github.io/react-strict-dom/learn/setup/#babel-configuration
RSD instructs users to configure babel.config.js which extract `api.caller(caller => caller && caller.platform)`, so this just uses passes the correct value.